internal/resources/syntheticmonitoring/resource_check.go

```diff
@@ -752,6 +924,14 @@ func ResourceCheckRead(ctx context.Context, d *schema.ResourceData, meta interfa
 		settings.Add(map[string]interface{}{
 			"traceroute": traceroute,
 		})
+	case chk.Settings.Multihttp != nil:
+		multiHttp := schema.NewSet(
```

Comment:
I'm pretty sure something more than this has to happen here, but I'm not sure what